catalog/imaging/pom.xml

```diff
@@ -28,7 +28,7 @@
     <name>Alliance :: Imaging</name>
 
     <properties>
-        <nitf-imaging.version>0.5</nitf-imaging.version>
+        <nitf-imaging.version>0.6-SNAPSHOT</nitf-imaging.version>
```

Comment:
We need to wait for a release of this.  We have enough snapshot versions - I would keep this ticket open until `0.6` is released.